lightning-invoice/src/payment.rs

```diff
@@ -78,10 +81,15 @@
 //! #
 //! # struct FakeRouter {}
 //! # impl Router for FakeRouter {
-//! #     fn find_route<S: Score>(
-//! #         &self, payer: &PublicKey, params: &RouteParameters, payment_hash: &PaymentHash,
-//! #         first_hops: Option<&[&ChannelDetails]>, scorer: &S
-//! #     ) -> Result<Route, LightningError> { unimplemented!() }
+//! # 	fn find_route(
+//! # 		&self, payer: &PublicKey, params: &RouteParameters, payment_hash: &PaymentHash,
+//! # 		first_hops: Option<&[&ChannelDetails]>, _inflight_htlcs: InFlightHtlcs
+//! # 	) -> Result<Route, LightningError> { unimplemented!() }
+//! #
+//! #   fn notify_payment_path_failed(&self, path: Vec<&RouteHop>, short_channel_id: u64) {  unimplemented!() }
+//! # 	fn notify_payment_path_successful(&self, path: Vec<&RouteHop>) {  unimplemented!() }
+//! # 	fn notify_payment_probe_successful(&self, path: Vec<&RouteHop>) {  unimplemented!() }
+//! # 	fn notify_payment_probe_failed(&self, path: Vec<&RouteHop>, short_channel_id: u64) { unimplemented!() }
```

Comment:
Similarly here. One space after `#` and four spaces to indent.

Reply:
Addresed in c5206b3bf646b61db957ef7dd80b79538114181d